src/mrack/providers/openstack.py

```diff
@@ -301,6 +304,13 @@ def get_network(self, name=None, ref=None):
         network = self.networks.get(name)
         if not network:
             network = self.networks_by_ref.get(ref)
+
+        if not network:
+            net_id = name or ref
+            logger.warning(
+                f"{self.dsp_name}: Failed to load network " f"with name: '{net_id}'"
```

Comment:
There is `" f"` in the middle of the line.  

D you really want to use "warning" for this?

Reply:
black must have messed up thanks for the note